src/components/pages/Planning/Planning.component.tsx

```diff
@@ -1,266 +1,157 @@
     name: string;
-    ownerID: number;
-    status: StoryStatus;
-    priority?: Priority;
-    creatorID: number;
-    createdAt: Date;
-    updatedAt?: Date;
+    onClick?: () => void;
 }
 
-const phases: Phase[] = [
-    {
-        id: 1,
-        name: 'Discovery',
-        status: 'TODO',
-        expectedStartAt: moment('2024-03-01', 'YYYY-MM-DD').toDate(),
-        expectedEndAt: moment('2024-03-20', 'YYYY-MM-DD').toDate(),
-        creatorID: 1,
-        totalStories: 50,
-        deliveredStories: 6,
-        createdAt: new Date(),
-    },
-    {
-        id: 2,
-        name: 'Define & Hypothesize',
-        status: 'TODO',
-        expectedStartAt: moment('2024-03-07', 'YYYY-MM-DD').toDate(),
-        expectedEndAt: moment('2024-03-18', 'YYYY-MM-DD').toDate(),
-        creatorID: 1,
-        totalStories: 60,
-        deliveredStories: 10,
-        createdAt: new Date(),
-    },
-    {
-        id: 3,
-        name: 'Prototyping',
-        status: 'TODO',
-        expectedStartAt: moment('2024-03-15', 'YYYY-MM-DD').toDate(),
-        expectedEndAt: moment('2024-03-20', 'YYYY-MM-DD').toDate(),
-        creatorID: 1,
-        totalStories: 70,
-        deliveredStories: 15,
-        createdAt: new Date(),
-    },
-    {
-        id: 4,
-        name: 'Visual Design',
-        status: 'TODO',
-        expectedStartAt: moment('2024-03-04', 'YYYY-MM-DD').toDate(),
-        expectedEndAt: moment('2024-03-08', 'YYYY-MM-DD').toDate(),
-        creatorID: 1,
-        totalStories: 80,
-        deliveredStories: 20,
-        createdAt: new Date(),
-    },
-    {
-        id: 5,
-        name: 'Development',
-        status: 'TODO',
-        expectedStartAt: moment('2024-03-12', 'YYYY-MM-DD').toDate(),
-        expectedEndAt: moment('2024-03-15', 'YYYY-MM-DD').toDate(),
-        creatorID: 1,
-        totalStories: 90,
-        deliveredStories: 25,
-        createdAt: new Date(),
-    },
-    {
-        id: 6,
-        name: 'Launch',
-        status: 'TODO',
-        expectedStartAt: moment('2024-03-18', 'YYYY-MM-DD').toDate(),
-        expectedEndAt: moment('2024-03-22', 'YYYY-MM-DD').toDate(),
-        creatorID: 1,
-        totalStories: 100,
-        deliveredStories: 30,
-        createdAt: new Date(),
-    },
-];
+export function PlanningComponent(props: Props) {
+    const currentTeam = props.deps.graphSource.currentTeam();
+    if (!currentTeam) {
+        return null;
+    }
+    // TODO: intercept the route and get the project, phase, and story
+    const routeUpdateParams = props.deps.router.currentRouteUpdate.params;
+    const projectId = toInt(getProjectId(routeUpdateParams));
+    const phaseId = toInt(getPhaseId(routeUpdateParams));
+    const storyId = toInt(getStoryId(routeUpdateParams));
 
-interface Level {
-    name: string;
-    onClick: () => void;
-}
+    const project = projectId && props.deps.graphSource.project(projectId);
+    const phase = phaseId ? props.deps.graphSource.phase(phaseId) : undefined;
```

Comment:
```suggestion
    const phase = phaseId ? props.deps.graphSource.phase(phaseId) : undefined;
```
Do we need to use `?` instead of `&&`.